subprojects/test-kit/src/main/java/org/gradle/testkit/runner/internal/DefaultGradleRunner.java

```diff
@@ -276,6 +276,12 @@ public BuildResult buildAndFail() {
         });
     }
 
+    @Override
+    public BuildResult run() throws InvalidRunnerConfigurationException {
```

Comment:
Seems odd we're adding a throws clause for `InvalidRunnerConfigurationException`, which is a runtime exception. Especially given that `build` and `buildAndFail` do not declare the same clause. Is there a reason this is here? I don't usually come across `throws RuntimeException` in the wild very often. 

LGTM otherwise

Reply:
@melix If you want this in 8.0, please respond ASAP. We're actively pruning PRs and issues from 8.0 due to the upcoming release.

Reply:
mmm, you're correct, I can't remember why I added this exception :thinking: 

I have rebased onto `release` and removed the `throws` clause.